packages/@lwc/template-compiler/src/parser/attribute.ts

```diff
@@ -39,13 +39,13 @@ import {
 } from './constants';
 
 function isQuotedAttribute(rawAttribute: string) {
-    const [, value] = rawAttribute.split('=');
-    return value && value.startsWith('"') && value.endsWith('"');
+    const attrVal = rawAttribute.slice(rawAttribute.indexOf('=') + 1);
```

Comment:
Instead of these two functions both calling `rawAttribute.slice(rawAttribute.indexOf('=') + 1)`, the caller can get the `attrValue` and pass it as an param instead of the `rawAttribute`